accession-commons-core/src/main/java/uk/ac/ebi/ampt2d/commons/accession/core/AccessioningService.java

```diff
@@ -93,8 +114,8 @@
     /**
      * Deprecates an accession
      *
-     * @param accession
-     * @param reason
+     * @param accession Accession of the object that identify a particular model
+     * @param reason comment or the necessity of deprecation
```

Comment:
`the reason for deprecation` seems to be precisely what it is and more straightforward. `comment or the necessity` makes me think, confused and worried.